src/urllib3/contrib/securetransport.py

```diff
@@ -374,6 +376,19 @@ def _set_ciphers(self):
         )
         _assert_no_error(result)
 
+    def _set_alpn_protocols(self, protocols):
```

Comment:
Would be great to confirm this works via capturing a TLS handshake on a supported platform, have we tried that yet?

Reply:
There is a test in ```TestALPN::test_alpn_protocol_in_first_request_packet```, is that what you meant?

Reply:
I meant it'd be great to have an indication this all works on a live macOS 10.12+ system by verifying manually w/ Wireshark. I don't have a macOS machine so I can't help here but we can do that outside this PR though :)